service/db.hpp

```diff
@@ -200,6 +231,38 @@ namespace raven
             std::filesystem::remove(std::filesystem::current_path() / "albinos_service_test.db");
         }
     }
+
+    TEST_CASE_CLASS ("setting update db")
+    {
+        SUBCASE("setting update unknown id") {
+            config_db db{std::filesystem::current_path() / "albinos_service_test.db"};
+            setting_update setting_update_data{config_id_st{42}, {{"foo", "bar"}}};
+            CHECK_EQ(static_cast<short>(db.settings_update(setting_update_data)),
+                static_cast<short>(request_state::unknown_id));
+            std::filesystem::remove(std::filesystem::current_path() / "albinos_service_test.db");
+        }
+
+        SUBCASE("regular update setting") {
```

Comment:
```suggestion
        SUBCASE("normal update setting") {
```
Unless you meant regular as in repeated every X.